CHANGELOG.md

```diff
@@ -28,6 +28,11 @@ and this project adheres to
 
 ### Changed
 
+- Update CourseSerializer to bind order and enrollment related to the user
 - Refactor models to allow enrollment without order
 
+### Fixed
+
+- Set id address field as read-only
```

Comment:
```suggestion
- Set address "id" field as read-only
```

Reply:
I prefer to remove the entry as there is no release yet.